config.yaml

```diff
@@ -17,3 +17,48 @@ options:
 
       It is strongly recommended that you use 'juju trust' instead, if available.
       since config values can be read by anyone with read access to the model.
+    type: string
+    default: ''
+  install_sources:
```

Comment:
These fields have to go in `config.yaml` not `layer.yaml`, which makes sense, in case the operator wants to override the apt sources.